tests/scripts/ghcr_prune/test_helpers.py

```
@@ -0,0 +1,253 @@
+        def raise_for_status(self) -> None:  # pragma: no cover
+            return
+
+        def json(self) -> list[dict[str, Any]]:
+            return self._data
+
+    class DummyClient:
+        def get(
+            self,
+            url: str,
+            *,
+            headers: Mapping[str, str] | None = None,
```

Ruff (0.15.12)

[warning] Unused method argument: `headers`

(ARG002)